ui/lib/apps/SlowQuery/components/Detail.tsx

```diff
@@ -136,30 +131,32 @@ function DetailPage() {
                 </Expand>
               </Descriptions.Item>
             </Descriptions>
+            <CardTabs animated={false}>
+              <CardTabs.TabPane
+                tab={t('slow_query.detail.tabs.basic')}
+                key="basic"
+              >
+                <TabBasic data={data} />
+              </CardTabs.TabPane>
+              <CardTabs.TabPane
+                tab={t('slow_query.detail.tabs.time')}
+                key="time"
+              >
+                <TabTime data={data} />
+              </CardTabs.TabPane>
+              <CardTabs.TabPane
+                tab={t('slow_query.detail.tabs.copr')}
+                key="copr"
+              >
+                <TabCopr data={data} />
+              </CardTabs.TabPane>
+              <CardTabs.TabPane tab={t('slow_query.detail.tabs.txn')} key="txn">
+                <TabTxn data={data} />
+              </CardTabs.TabPane>
+            </CardTabs>
           </>
         )}
       </Head>
```

Comment:
All components in `<Head>` element?

Reply:
It depends on whether or not there should be paddings.